File: tests/golden/ts/src/sample.ts
const STATUS_CODE = "777";

@Component("777")
class Sample {
    check(input: string): number {
        let localCode = "777";
        if (input === "777") {
            return 1;
        }
        logValue("777");
        return -1;
    }

    getCode(): string {
        return "777";
    }

    // "777" のコメント — その他に分類されることを期待
}

File: tests/golden/kotlin/src/Sample.kt
package sample

object Constants {
    const val STATUS_CODE = "777"
}

@Deprecated("777")
class Sample {
    fun check(input: String): Int {
        var localCode = "777"
        if (input == "777") {
            return 1
        }
        if (input == Constants.STATUS_CODE) {
            return 0
        }
        logValue("777")
        return -1
    }

    fun getCode(): String {
        return "777"
    }

    // "777" のコメント — その他に分類されることを期待
}
